server/punishments.js

```diff
@@ -66,11 +66,11 @@ class PunishmentMap extends Map {
 	/**
 	 * @param {(punishment: Punishment, id: string, map: PunishmentMap) => void} callback
 	 */
-	forEach(callback) {
-		super.forEach((punishment, k) => {
+	forOf(callback) {
```

Comment:
lol

This is actually something to watch out for; you might even want to implement this as an iterator.

But if you don't, I'd just keep this unrenamed.